.github/workflows/gradle-build.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow uses actions that are not certified by GitHub.
```

Comment:
Is the goal here to run GHA instead of using Travis?
In any case this change seems orthogonal to the rest of the PR.

Reply:
I created that because I don't have a Travis or Jenkins environment to build this in and  my personal laptop has an M1 chip in it. Additionally, I wanted to be sure that I had fixed the issue before I submitted the PR but left it in because I felt it could be beneficial to others who might want to build the project but don't have much Jenkins, Travis, or Gradle experience.

The action only runs when triggered manually.

Reply:
there are some unittests for es in `tests/src/test/scala/org/apache/openwhisk/core/database/elasticsearch/ElasticSearchActivationStoreTests.scala` which is using `testcontainers`, so I think you can use it to run tests on your laptop?

Reply:
It failed when trying to download one of the dependencies. I'll try again so I can get the dependency name.